packages/codegen/src/generateMetadataFile.ts

```diff
@@ -110,16 +110,9 @@ function generateFields(config: Config, dbMetadata: EntityDbMetadata): Record<st
       } else {
         serdeType = PrimitiveSerde;
       }
-      if (serdeType === PrimitiveSerde || serdeType === DateSerde) {
```

Comment:
Given `timeZone` is held in this new global `runtimeConfig`, we don't have to pass it into the individual serde constructors.